slidechain/import.go

```diff
@@ -128,7 +128,7 @@ func (c *Custodian) doImport(ctx context.Context, nonceHash []byte, amount int64
 		return errors.Wrap(err, "computing transaction ID")
 	}
 	importTx.Runlimit = math.MaxInt64 - runlimit
-	_, err = c.S.submitTx(ctx, importTx)
+	_, err = c.S.submitTx(ctx, importTx, defaultBlockInterval)
```

Comment:
Would we want to add this add a field on the `custodian` or the `submitter` so that we can configure it as a flag in `slidechaind`?